submissions/ismailtlem/gblabla_coding_challenge.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any
+import numpy as np
```

Comment:
Is that available in standard library ? Unfortunately no additional library should be used